spec/views/instructeur/shared/avis/list.html.haml_spec.rb

```diff
@@ -26,4 +29,10 @@
       expect(subject).to include(simple_format(avis.first.answer))
     end
   end
+
+  context 'with another instructeur' do
+    let(:avis) { [create(:avis, :with_answer, claimant: instructeur2, experts_procedure: experts_procedure, introduction_file: introduction_file)] }
+
+    it { is_expected.to have_text("Fichier d'introduction à l'avis fourni par #{avis.first.claimant.email}") }
```

Comment:
Je dirais qu'on peut mettre écrire explicitement dans cette spec le comportement attendu :

 ```suggestion
    it 'shows the files attached to the avis request' do
      expect(subject).to have_text("Fichier d'introduction à l'avis fourni par #{avis.first.claimant.email}")
    end
```